packages/core/src/sync-realtime/index.ts

```diff
@@ -643,15 +758,15 @@ export const createRealtimeSync = (
       }
     }
 
-    callTraces = callTraces.filter(
-      (trace) => revertedTransactions.has(trace.transactionHash) === false,
-    );
+    // callTraces = callTraces.filter(
+    //   (trace) => revertedTransactions.has(trace.transactionHash) === false,
+    // );
 
     return {
       block,
       logs,
       factoryLogs,
-      callTraces,
+      traces,
```

Comment:
Should this type be the same as ?? https://github.com/ponder-sh/ponder/blob/2117cec2f8e477d5446ab0242b07a82d054f50e1/packages/core/src/sync-historical/index.ts#L313-L317